src/js/AutoshareForTwitterPrePublishPanel.js

```diff
@@ -31,7 +31,10 @@ class AutoshareForTwitterPrePublishPanel extends Component {
 
```

Comment:
Changes in this file came out of WP Acceptance testing. It's helpful to disable the submit button while the plugin's API request is running so that the post is not submitted before the autoshare metadata has been saved.